src/FileUpdater.js

```diff
@@ -68,14 +67,9 @@ function updatePathWithStats (sourcePath, sourceStats, targetPath, targetStats,
 
         if (targetStats) {
             // The target exists. But if the directory status doesn't match the source, delete it.
-            if (targetStats.isDirectory() && !sourceStats.isDirectory()) {
-                log('rmdir  ' + targetPath + ' (source is a file)');
-                shell.rm('-rf', targetFullPath);
-                targetStats = null;
-                updated = true;
-            } else if (!targetStats.isDirectory() && sourceStats.isDirectory()) {
-                log('delete ' + targetPath + ' (source is a directory)');
-                shell.rm('-f', targetFullPath);
+            if ((targetStats.isDirectory() && !sourceStats.isDirectory()) || (!targetStats.isDirectory() && sourceStats.isDirectory())) {
```

Comment:
Condition is equivalent to `targetStats.isDirectory() !== sourceStats.isDirectory()`.
I'd merge with containing block to `targetStats && (targetStats.isDirectory() !== sourceStats.isDirectory())`